client/blocks/comment-detail/comment-detail-author.jsx

```diff
@@ -135,7 +140,7 @@ export class CommentDetailAuthor extends Component {
 								{ authorDisplayName }
 							</strong>
 							<span>
-								{ authorUrl }
+								{ withoutHttp( authorUrl ).replace( /\/$/, '' ) }
```

Comment:
from Slack: how about just parsing the URL and making it hear from parts…

```js
const authorLocation = parseUrl( authorUrl );

{ ${ authorLocation.hostname }/${ authorLocation.path } }
```

something like that